src/lib/modules/charts/components/extra-params/percentile-historic.component.ts

```diff
@@ -48,7 +49,7 @@ export class PercentileHistoricComponent implements AfterViewInit, OnInit {
     createForm() {
         this.percentileHistoricForm = this.formBuilder.group({
             historicCtl: [this.extraParams.historic_range || this.defaultHistoric],
-            percentileCtl: [this.extraParams.percentile || this.defaultPercentile, Validators.required]
+            percentileCtl: [this.extraParams.percentile || HistoricPercentileParam.NinetyFive]
```

Comment:
Awk. The API has a different default for different extreme indicators. 1, 99, and 99. I'm not so concerned about matching _every_ case, but maybe let's do `NinetyNine`.

https://docs.climate.azavea.com/indicators.html#extreme-cold-events

Reply:
Oh. Yeah that isn't great. I'll see what I can do to clean this up.